src/tests/ftest/pool/create_test.py

```diff
@@ -0,0 +1,314 @@
+            Create 100 pools on all of the servers.
+            Perform an orderly system shutdown via cmd line (dmg).
+            Restart the system via cmd line tool (dmg).
+            Verify that DAOS is ready to accept requests with in 2 minutes.
+
+        :avocado: tags=all,pr,hw,large,pool,create_performance
+        """
+        # Create some number of pools each using a equal amount of 60% of the
+        # available capacity, e.g. 0.6% for 100 pools.
+        quantity = self.params.get("quantity", "/run/pool/*", 1)
+        ratio = 0.6 / quantity
+        self.define_pools(quantity, ratio, ratio)
+        self.check_pool_creation(3)
+
+        # Verify DAOS can be restarted in less than 2 minutes
+        try:
+            self.server_managers[0].system_stop()
+        except ServerFailed as error:
+            self.fail(error)
+
+        start = float(time.time())
+        try:
+            self.server_managers[0].system_start()
+        except ServerFailed as error:
+            self.fail(error)
+
+        duration = float(time.time()) - start
+        self.assertLessEqual(
+            duration, 120,
+            "DAOS not ready to accept requests with in 2 minutes")
+
+        # Verify all the pools exists after the restart
+        pool_uuid_list = [pool.uuid for pool in self.pool]
+        result = self.dmg.get_output("pool_list")
+        self.assertListEqual(
+            pool_uuid_list, result,
+            "Pool UUID list does not match after reboot")
+
+    @skipForTicket("DAOS-5203")
+    def test_create_no_space(self):
+        """JIRA ID: DAOS-3728.
+
+        Test Description:
+            Create a pool using most of the capacity of one server on only one
+            server.  Verify that attempting to create  a pool of the same size
+            across all of the servers fails due to no space.  Now verify that
+            creating a pool of the same size on across all but the first pool
+            succeeds.
+
+        :avocado: tags=all,pr,hw,large,pool,create_no_space
+        """
+        # Define three pools to create:
+        #   - one pool using 90% of the available capacity of one server
+        #   - one pool using 90% of the available capacity of all servers
+        #   - one pool using 90% of the available capacity of the other server
+        self.define_pools(3, 0.9, 0.9)
+        ranks = [rank for rank, _ in enumerate(self.hostlist_servers)]
+        self.pool[0].target_list.update(ranks[:1], "pool[0].target_list")
+        self.pool[1].target_list.update(ranks, "pool[1].target_list")
+        self.pool[2].target_list.update(ranks[1:], "pool[2].target_list")
+
+        # Disable failing the test if a pool create fails
+        self.dmg.exit_status_exception = False
+
+        # Create all three pools.  The creation of the first and third pools
+        # should succeed.  The second pool creation should fail due to not
+        # enough space.
+        self.pool[0].create()
+        self.assertTrue(
```

Comment:
This is fine, doesn't have to be changed. But why not use `self.assertEqual` instead?

Reply:
That's a valid point.  The `assertEqual` call would probably make more sense in this case.